library/core/src/ptr/const_ptr.rs

```diff
@@ -725,8 +725,9 @@ impl<T: ?Sized> *const T {
     ///
     /// [`ptr::read`]: crate::ptr::read()
     #[stable(feature = "pointer_methods", since = "1.26.0")]
+    #[rustc_const_unstable(feature = "const_ptr_read", issue = "none")]
```

Comment:
we should probably create a tracking isssue for this feature, can you open one referencing this PR as the initial impl PR and just mentioning some of the caveats that were necessary to get this to work?

Reply:
I have opened #80377. Is that how it should look? :)

Reply:
That looks great!